tests/test_replication.py

```diff
@@ -34,7 +34,7 @@ def test_replication_behavior(self):
             ('BF.ADD', 'BF.ADD key item', 'BF.ADD key item1', 2),
             ('BF.MADD', 'BF.MADD key item', 'BF.MADD key item1', 2),
             ('BF.RESERVE', 'BF.RESERVE key 0.001 100000', 'BF.ADD key item1', 1),
-            ('BF.INSERT', 'BF.INSERT key items item', 'BF.INSERT key items item1', 2),
+            ('BF.INSERT', 'BF.INSERT key items item tightening 0.5', 'BF.INSERT key items item1 tightening 0.5', 2),
```

Comment:
```suggestion
            ('BF.INSERT', 'BF.INSERT key items item', 'BF.INSERT key items item1', 2),
```

This is an invalid test. The tightening ratio is coming after `ITEM` so it is treated as an item. It is not treated as a property. Also, we cannot use this command on the primary, since we only allow this arg during replication :D 

Let's just use a normal BF.INSERT command.

Once we support the tightening ratio config, we can then (1) update the config on the primary node, (2) Use BF.INSERT (3) Validate Digest matches on the replica even though the tightening ratio config is different

For now, let us leave it as it is because it is essentially untested code (unless we use a monitor on replica during bloom creation on the primary node)

Reply:
Ah got it, will revert this change accordingly